en/apispec/openapi.json

```diff
@@ -3218,199 +2617,2447 @@
+              }
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "headers": {
+              "Qtl-Request-Id": {
+                "schema": {
+                  "type": "string"
+                },
+                "description": "A string automatically generated by the API server to identify each request."
+              }
+            },
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "code": {
+                      "$ref": "#/components/schemas/response.code"
+                    },
+                    "message": {
+                      "$ref": "#/components/schemas/response.message"
+                    },
+                    "status": {
+                      "$ref": "#/components/schemas/response.status"
+                    }
+                  }
+                },
+                "examples": {
+                  "AccessDenied": {
+                    "value": {
+                      "code": "AccessDenied",
+                      "message": "A regular customer is not allowed to make this call.",
+                      "status": 0
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "get-serviceQuotas",
+        "description": "The API allows a reseller to get a list of service quotas of its child customers.\nBoth \"servicequotas\" and \"serviceQuotas\" are accepted in the path, which also applies to other APIs related to serviceQuotas.\n\n",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/Authorization"
+          },
+          {
+            "$ref": "#/components/parameters/Date"
+          },
+          {
+            "$ref": "#/components/parameters/Accept-Encoding"
+          },
+          {
+            "$ref": "#/components/parameters/Accept"
+          },
+          {
+            "$ref": "#/components/parameters/On-Behalf-Of"
+          },
+          {
+            "$ref": "#/components/parameters/Content-Type"
+          },
+          {
+            "schema": {
+              "type": "string",
+              "enum": [
+                "Trial",
+                "Active",
+                "Suspended"
+              ]
+            },
+            "in": "query",
+            "name": "status",
+            "description": "Status of the customer with respect to HDT service. "
+          },
+          {
+            "schema": {
+              "type": "string"
+            },
+            "in": "query",
+            "name": "contractID",
+            "description": "ID of the contract signed for HDT service subscription.   For self-signup customers, we’ll use some default contractIDs (We might have different contractIDs for different payment currencies). This query parameter can be used to filter such customers."
```

Comment:
I'm not sure why you need to include the 2nd, 3rd, and 4th sentences describing our internal use of the field.

Reply:
Thanks, I’ll look into your comments.
And for API doc, we will also make a significant change soon after the new billing method API is completed.